website/docs/r/x509_authentication_database_user.html.markdown

```diff
@@ -17,6 +17,8 @@ description: |-
 
 -> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
 
+-> **NOTE:** Before provider version 1.14.0, Self-managed X.509 Authentication was disabled for the project when this resource was deleted.
```

Comment:
This feels incomplete. Should we clarify that now we do nothing and why?

Reply:
thx, added more info